e2e/buttons.e2e-spec.ts

```diff
@@ -0,0 +1,77 @@
+import {
+  browser,
+  by,
+  element
+} from 'protractor';
+
+import {
+  expect,
+  SkyHostBrowser
+} from '@skyux-sdk/e2e';
+
+describe('Buttons', () => {
+  function hoverMouseOverSelector(selector: string): any {
+    return browser.actions()
+      .mouseMove(element(by.css(`#screenshot-buttons-default-state ${selector}`)))
+      .perform();
+  }
+
+  beforeEach(() => {
+    SkyHostBrowser.get('visual/buttons');
+    SkyHostBrowser.setWindowBreakpoint('md');
```

Comment:
I'm assuming we don't need extra breakpoints, because the buttons don't really have different states?

Reply:
True. Here, I'm just setting the breakpoint to `md` for every test to make sure the screenshot's bounds are consistent.